.github/workflows/create-release.yml

```diff
@@ -51,7 +51,7 @@ jobs:
           submodules: true
 
       - name: update-version
-        uses: KSPModdingLibs/KSPBuildTools/.github/actions/update-version@main
+        uses: ./KSPBuildTools/.github/actions/update-version
```

Comment:
can we revert these?  build.yml uses @main so this seems inconsistent.  We need to find a way to pin versions, but if this isn't the final solution then let's just leave it out of this PR.